framework/wazuh/exception.py

```diff
@@ -138,6 +138,8 @@ class WazuhException(Exception):
         1742: 'Error running XML syntax validator',
         1743: 'Error running Wazuh syntax validator',
         1744: 'Invalid chunk size',
+        1745: "Agent only belongs to 'default' and it cannot be unset from this group.",
+        1746: "Agent doesn't belong to any group",
```

Comment:
I have the doubt between these two messages:
* Agent doesn't belong to any group
* Agent belongs to default group only and cannot be removed

Reply:
The exception `1746`  is raised when the agent doesn't belong to any group, it's the first check the function does.
The exception `1745` is raised when the function receives `default` as `group_id` argument. This exception is raised when the agent belongs to any group.

Reply:
I have finally removed exception `1746` in commit https://github.com/wazuh/wazuh/pull/2838/commits/2ddf6b7e6c6a0df82fab94dedfbb85b4c7abe731.